src/pages/Club/Club/Club.tsx

```diff
@@ -215,28 +203,29 @@ const Club = () => {
   }
 
   const addClubAdmin = async (admin: ClubAdmin) => {
+    let previousAdmin: ClubAdmin = new ClubAdmin();
+    admins.forEach(_admin => {
+      if(_admin.adminType.adminTypeName == admin.adminType.adminTypeName){
```

Comment:
Dont think this '_' is a suitable notation for the typscript code. We dont use it anywhere else

Reply:
Should i use the "element" instead?

Reply:
can be "clubAdmin" or "currentAdmin" or anything like that

Reply:
Got it, will fix with the next commit

Reply:
Done!